v03_pipeline/lib/tasks/write_imported_callset.py

```diff
@@ -85,6 +86,7 @@ def create_table(self) -> hl.MatrixTable:
         mt = select_relevant_fields(mt, self.dataset_type)
         if self.dataset_type.has_multi_allelic_variants:
             mt = split_multi_hts(mt)
+        mt = annotate_vets(mt)
```

Comment:
VETS is allele-specific, so it needs to be ran on a split MT , which it looks like is covered here. And if the prior 'if' statement isn't triggered, then it wouldn't be a problem... I don't think?

Reply:
yep, this logic was placed explicitly after the `split_multi`.  The `if` statement will always be true for `SNV_INDEL`.  it might be even clearer to slot `annotate_vets` under an `if` statement as well, but the function will be a no-op if the `info` field is missing.

Reply:
I think there should be a comment here explaining why we are adding this one specific annotation here when this function otherwise does not do that. Alternatively, maybe make it a dataset-type-specific, generically named function, something like `annotate_imported_mt`